src/os/bluestore/NVMEDevice.cc

```diff
@@ -810,18 +843,57 @@ static void write_split(
     bl.splice(0, write_size, &t->bl);
     remain_len -= write_size;
     t->ctx = ioc;
-    first = static_cast<Task*>(ioc->nvme_task_first);
-    last = static_cast<Task*>(ioc->nvme_task_last);
-    if (last)
-      last->next = t;
-    if (!first)
-      ioc->nvme_task_first = t;
-    ioc->nvme_task_last = t;
-    ++ioc->num_pending;
+    ioc_task_update(ioc, t);
     begin += write_size;
   }
 }
 
+static void read_split(
+    NVMEDevice *dev,
+    uint64_t aligned_off,
+    IOContext *ioc, char *buf, uint64_t aligned_len, Task *primary,
+    uint64_t orig_off, uint64_t orig_len)
+{
+  uint64_t remain_aligned_len = aligned_len, begin = 0, read_size;
```

Comment:
please initialize the local variables in the declaration. see https://google.github.io/styleguide/cppguide.html#Local_Variables

Reply:
done